packages/query-core/src/__tests__/streamedQuery.test.tsx

```diff
@@ -350,14 +350,18 @@ describe('streamedQuery', () => {
     unsubscribe()
   })
 
-  test('should support maxChunks', async () => {
+  test('should support custom reducer', async () => {
     const key = queryKey()
+
     const observer = new QueryObserver(queryClient, {
       queryKey: key,
-      queryFn: streamedQuery({
-        queryFn: () => createAsyncNumberGenerator(3),
-        maxChunks: 2,
-      }),
+      queryFn: streamedQuery<number, Record<number, boolean>>({
+        queryFn: () => createAsyncNumberGenerator(2),
+        reducer: (acc, chunk) => ({
+            ...acc,
```

Comment:
the first time `reducer` is invoked, `acc` is an empty array, because `placeholderData` is not specified.
it seems a bit strange to me. but i'm not sure how to solve it. 
i can apply the same logic as in native `reduce`... 
 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#initialvalue.